saas/api/roles.py

```diff
@@ -723,13 +722,34 @@ class RoleFilteredListAPIView(RoleSmartListMixin, RoleByDescrQuerysetMixin,
     """
     serializer_class = RoleSerializer
 
+    def get(self, request, *args, **kwargs):
+        response = super(RoleFilteredListAPIView, self).get(
+            request, *args, **kwargs)
+        data = {
```

Comment:
No! DRF already provides a framework to implement this kind of decorators.
This should be implemented as special paginator (see for example [StatementBalancePagination](https://github.com/djaodjin/djaodjin-saas/blob/76190d48122b0f196b935136c2f1505cceefc33b/saas/api/transactions.py#L47))

Reply:
I didn't know what was the best approach and that I can use a custom paginator here.